README.md

```diff
@@ -1018,8 +1018,36 @@ validate threshold            Validate the compliance and status counts of an HD
 
 ### Generate
 
+#### Update Controls
+Use this process prior of running the `Delta` if the updated guidance's have new control numbers. Running this process minimizes the delta output content and makes for better and easier visualization of the modification provided by the Delta process.
+
+```
+USAGE
+  $ saf generate update_controls -X <value> -J <value> -c <value> [-P V|VS] [--[no-]backupControls] [--[no-]formatControls] [-L info|warn|debug|verbose]
+
+FLAGS
+  -h, --help                    Show CLI help.
+  -X, --xccdfXmlFile=<value>    (required) The XCCDF XML file containing the new guidance - in the form of .xml file
```

Comment:
Should we make this optional if individuals want to format but not change any control numbers? This would impact logic as well.